### PR TITLE
Refactor Event-visitors.

### PR DESCRIPTION
**Pull request by Depesi**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress-Frontend/projects/1)

## Code reviewers
Vlad Babyak
Vadym Kharyton
- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue
Сlass component
Code duplication

## Summary of change
Refactor to func component
Created a function thanks to which there is no code duplication

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by VadymKharyton — COMMENTED** [review]
Changes look good. Only one point: please clean code from redundant `console.log` calls and empty comments statements, then will approve

File: src/components/event/EventItem/Event-item.js [hunk 1]
```diff
@@ -63,10 +63,10 @@ export default class EventCard extends Component {
             eventStatus,
             categories,
             countVisitor,
-            owners,
+            organizers: owners,
             members,
         } = this.props.item;
-        
+        console.log(this.props.item);
```
Comment (andrii-kuzmichov) [inline src/components/event/EventItem/Event-item.js:69]: Clean up

File: src/components/event/EventItem/Event-item.js [hunk 2]
```diff
@@ -63,10 +63,10 @@ export default class EventCard extends Component {
             eventStatus,
             categories,
             countVisitor,
-            owners,
+            organizers: owners,
```
Comment (andrii-kuzmichov) [inline src/components/event/EventItem/Event-item.js:66]: Add todo

File: src/components/landing/CarouselEventCard.js [hunk 3]
```diff
@@ -29,15 +29,18 @@ export default class CarouselEventCard extends Component {
     }
 
     render() {
+
         const event = this.props.event
+		console.log(event);
```
Comment (andrii-kuzmichov) [inline src/components/landing/CarouselEventCard.js:34]: clean up

File: src/components/landing/CarouselEventCard.js [hunk 4]
```diff
@@ -29,15 +29,18 @@ export default class CarouselEventCard extends Component {
...
         return (
             <div className="card">
                 <img className="card-img-top" src={this.state.eventImage}
                     alt="Event image" />
                 <div className="card-body">
                     <p className="card-text text-muted">{parseDate(event.dateFrom)}</p>
                     <p className="card-text">{event.title}</p>
-                    <p className="card-text text-muted">{event.owners[0].username}</p>
+					{/*  */}
```
Comment (andrii-kuzmichov) [inline src/components/landing/CarouselEventCard.js:42]: Add todo coment

**Review by VadymKharyton — APPROVED** [review]

**Pull request closed without merging**